From 12098415cda75ed10d4d7c75602630b90ef6afd1 Mon Sep 17 00:00:00 2001
From: Hugh Waite <hugh.waite@acision.com>
Date: Fri, 22 May 2015 09:28:52 +0100
Subject: [PATCH] registrar: Fix insertion of ulattrs xavp into position 0

- If the first entry is an XTYPE_NULL entry it is replaced, not pushed down the stack
---
 modules/registrar/lookup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/modules/registrar/lookup.c
+++ b/modules/registrar/lookup.c
@@ -298,7 +298,7 @@
 
 		if(ptr->xavp!=NULL) {
 			xavp = xavp_clone_level_nodata(ptr->xavp);
-			if(xavp_add(xavp, NULL)<0) {
+			if(xavp_insert(xavp, 0, NULL)<0) {
 				ret = -3;
 				goto done;
 			}
